javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -150,6 +157,10 @@ public static int compareByName(AnnotationMirror a1, AnnotationMirror a2) {
       throw new BugInCF("Unexpected null argument:  compareByName(%s, %s)", a1, a2);
     }
 
+    // annotationName() contains a check for CheckerFrameworkAnnotationMirror, so the final `return`
+    // statement of this method should run about as fast as this `if` block.  Is the point of this
+    // block just the `if (name1 == name2) { return 0; }`, or is there some other justification for
```

Comment:
Yes the point is just that `name1 == name2` is faster than `name1.compareTo(name2)`.